src/fstests/fstests.jinja2

```diff
@@ -46,6 +48,40 @@ class Job(BaseJob):
             print('XML not found.', e)
             return 'fail'
 
+    def _parse_to_json(self, node, tree, name=None):
+        if 'result' in tree:
+            node['name'] = name
+            node['result'] = tree['result']
+        else:
+            if (not name == None): node['name'] = name
+            child_nodes = []
+            for name in tree:
+                child_nodes.append(self._parse_to_json({}, tree[name], name))
+            node['child_nodes'] = child_nodes
+        return node
+
+    def _parse_results(self, kdir):
+        xml_file = ET.parse(f"{kdir}/results.xml")
+        root = xml_file.getroot()
+        tree = {}
+        for test_case in root.iter('testcase'):
+            if test_case.find('skipped') != None:
```

Comment:
```
if test_case.find('skipped') is not None:
```
`None` is a singleton object in Python so the standard way to compare it with another object is with `is` / `is not`.